src/local_modules/common.iced

```diff
@@ -53,7 +54,25 @@ module.exports =
       prev.dep.unshift name
     
     # add the new task.
-    gulp.task name, deps, fn
+    # gulp.task name, deps, fn
+
+    if fn.length # see if the task function has arguments (betcha never saw that before!)
```

Comment:
once ran into having `f.length` instead of `f().length` with TSC not complaining, was mind-blown... and then forgot about it :-(